src/otx/tools/templates/keypoint_detection/rtmpose_tiny/template.yaml

```diff
@@ -0,0 +1,25 @@
+# Description.
+model_template_id: Keypoint_Detection_RTMPose_Tiny
+name: RTMPose-tiny
+task_type: KEYPOINT_DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: RTMPose-tiny is a lightweight keypoint detection model based on the RTMPose architecture. It is designed for fast inference and is suitable for edge devices.
+application: ~
+
+# Algo backend.
+framework: OTXKeypointDetection v1.0.0
+
+# Hyper Parameters
+hyper_parameters:
+  base_path: ../configuration.yaml
```

Comment:
Isn't it in the same directory?

Reply:
configuration.yaml located in the keypoint_detection/ folder, but template in the keypoint_detection/rtmpose_tiny. I made same structure as for other tasks